### DISCUSSION TITLE
如何修改默认的package下载位置

### DISCUSSION THREAD

**Discussion by freestriker** [xmake-io/xmake#3239] [Q&A] [answered]
C盘快满了，没有找的相关的设置默认下载路径的命令行参数，麻烦知道的朋友告知下，感谢

**Comment by waruqi** [comment, marked as answer]
https://xmake.io/#/zh-cn/guide/configuration?id=xmake_globaldir
https://xmake.io/#/zh-cn/guide/configuration?id=xmake_pkg_installdir
https://xmake.io/#/zh-cn/guide/configuration?id=xmake_pkg_cachedir
